NewKernel_d/include/CGAL/NewKernel_d/Cartesian_LA_functors.h

```diff
@@ -57,65 +57,65 @@ template<class R_,class Zero_> struct Construct_LA_vector
                 return std::move(v);
         }
         template<class...U>
-        typename std::enable_if<Constructible_from_each<RT,U...>::value &&
+        std::enable_if_t<Constructible_from_each<RT,U...>::value &&
                 std::is_same<Dimension_tag<int(sizeof...(U))>, Dimension>::value,
-          result_type>::type
+          result_type>
         operator()(U&&...u)const{
                 return typename Constructor::Values()(std::forward<U>(u)...);
         }
-        //template<class...U,class=typename std::enable_if<Constructible_from_each<RT,U...>::value>::type,class=typename std::enable_if<(sizeof...(U)==static_dim+1)>::type,class=void>
+        //template<class...U,class=std::enable_if_t<Constructible_from_each<RT,U...>::value>::type,class=typename std::enable_if_t<(sizeof...(U)==static_dim+1)>,class=void>
```

Comment:
```suggestion
        //template<class...U,class=std::enable_if_t<Constructible_from_each<RT,U...>::value>,class=typename std::enable_if_t<(sizeof...(U)==static_dim+1)>,class=void>
```
unless I am missing something? (I know it is just a comment, but mostly I am confused about how it happened because I thought you had used an automated tool to do the replacement)